### ISSUE TITLE
Rename vendored folder from "vendor/_nuts" to "internal"

### ISSUE THREAD

**Issue by owenthereal** [jingweno/nut#11]
Currently `nut` vendors source code to "vendor/_nuts". As discussed in [this thread](https://groups.google.com/forum/?pli=1#!topic/golang-dev/nMWoEAG55v8), let's standardize it to "internal". The following changes are needed:
- rename the folder to "internal"
- don't pull in Go tests for dependencies

The second changes will get `go test ./...` to work without running dependencies' tests.

**Comment by dvirsky** [comment]
why not let the user decide on the package name using a command line flag BTW?  you can default to `internal`

**Comment by owenthereal** [comment]
> why not let the user decide on the package name using a command line flag BTW?

As you may read from the discussion linked in the issue, we're trying to standardize the folder of the vendored dependencies across all vendoring tools. I'm afraid this is a hard requirement. The point is that other tools will still work.

**Comment by dvirsky** [comment]
I agree that this should be the default but there isn't really a reason not to let people override it if they have a good reason to do it. Anyway it's no big deal either way.

**Comment by dvirsky** [comment]
Did this in #15  - no config, just hard coded
